static/css/styles.css

```diff
@@ -434,6 +435,7 @@ ul.navbar-nav>li {
 .planner-week-top-content td {
 	font-size: 16px;
 	height: 40px;
+	border-bottom: 1px solid #fff;
```

Comment:
Try not to modify styles that other views rely on, can you make a duplicate style for now without the bottom border?